src/commands/ProjectCommand.ts

```diff
@@ -0,0 +1,60 @@
+import Command from "../abstracts/Command";
+import {Message, MessageEmbed} from "discord.js";
+// @ts-ignore
```

Comment:
Would you mind adding to this comment explaining why the `@ts-ignore` is needed?

Reply:
File '......' is not under 'rootDir' '...........'. 'rootDir' is expected to contain all source files.